src/plugins/filter/demo/post-filter.js

```diff
@@ -0,0 +1,17 @@
+( function() {
+window[ "wb-filter" ] = {
+	filterCallback: function( $field, $elm, settings ) {
+		var $sections =	$elm.find( "section" ),
```

Comment:
Better than a callback no. More WET4....maybe. However, the next version of WET needs to shift to using more callback as it's a way more common pattern in the JS world.